client/me/account/main.jsx

```diff
@@ -881,10 +882,13 @@ const Account = createReactClass( {
 			<Main className="account is-wide-layout">
 				<PageViewTracker path="/me/account" title="Me > Account Settings" />
 				<MeSidebarNavigation />
-				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
+				{ false && <ReauthRequired twoStepAuthorization={ twoStepAuthorization } /> }
 				<FormattedHeader brandFont headerText={ translate( 'Account Settings' ) } align="left" />
 
 				<Card className="account__settings">
+					<CardHeading className="account__settings-card-header" tagName="h2" size={ 14 }>
+						{ translate( 'Account Information' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Clear account information' )` *ES Score: 6*